scripts/update_python.py

```diff
@@ -24,15 +24,17 @@
 import glob
 import multiprocessing
 import os
+import platform
 import urllib.request
 import shutil
 import subprocess
 import sys
 from subprocess import check_call
 
-version = '3.7.4'
+version = '3.9.2'
+major_minor_version = '.'.join(version.split('.')[:2])  # e.g. '3.9.2' -> '3.9'
 base = 'https://www.python.org/ftp/python/%s/' % version
-revision = '2'
+revision = '1'
```

Comment:
Can we do this without bumping the version, or if this needed for M1 support?

Reply:
The old version does not build for M1, updating to latest version was necessary to build clean.

Reply:
sgtm